src/uint/encoding.rs

```diff
@@ -476,6 +498,252 @@ fn decode_str_radix_aligned_digits<D: DecodeByLimb>(
     Ok(())
 }
 
+#[cfg(feature = "alloc")]
+pub(crate) fn radix_encode_limbs_to_string(radix: u32, limbs: &[Limb]) -> String {
+    let mut array_buf = [Limb::ZERO; 128];
+    let mut vec_buf = Vec::new();
+    let limb_count = limbs.len();
+    let buf = if limb_count <= array_buf.len() {
+        array_buf[..limb_count].copy_from_slice(limbs);
+        &mut array_buf[..limb_count]
+    } else {
+        vec_buf.extend_from_slice(limbs);
+        &mut vec_buf[..limb_count]
+    };
+    radix_encode_limbs_mut_to_string(radix, buf)
+}
+
+#[cfg(feature = "alloc")]
+pub(crate) fn radix_encode_limbs_mut_to_string(radix: u32, limbs: &mut [Limb]) -> String {
+    if !(2..=26).contains(&radix) {
```

Comment:
The comment above says "Panics if `radix` is not in the range from 2 to 36."

Reply:
Oops, apparently I forgot halfway through what the upper limit was meant to be. Fixed now.

Reply:
Perhaps it could be made a named constant?